Jenkinsfile

```diff
@@ -48,7 +48,7 @@ spec:
   }
   parameters {
     string(name: 'GF_BUNDLE_URL', 
-           defaultValue: 'https://ci.eclipse.org/jakartaee-tck/job/build-glassfish/lastSuccessfulBuild/artifact/appserver/distributions/glassfish/target/glassfish.zip', 
+           defaultValue: 'https://download.eclipse.org/ee4j/glassfish/glassfish-7.0.0-SNAPSHOT-nightly.zip', 
            description: 'URL required for downloading GlassFish Full/Web profile bundle' )
     string(name: 'JAKARTA_INJECT_TCK_URL',
```

Comment:
@gurunrao should this instead be using https://download.eclipse.org/jakartaee/dependency-injection/2.0/jakarta.inject-tck-2.0.2-bin.zip?  This update of Dependency-injection works with CDI 4.0

Reply:
thanks scott, since PR is more than 3 months old, inject tck release yesterday was not captured.
I have updated the PR to reflect the same.